packages/zoe/src/zoe.js

```diff
@@ -683,6 +696,7 @@ const makeZoe = (additionalEndowments = {}) => {
               instanceHandle,
               proposal,
               currentAllocation: arrayToObj(amountsArray, userKeywords),
+              notifier: produceNotifier(),
```

Comment:
Calling this property `notifier` is now confusing because it holds a record with a property named `notifier`.

`notifierRec`?

Reply:
Probably better to flatten and add both `notifier` and `updater` directly.

Reply:
fixed.